bin/convection

```diff
@@ -74,29 +81,83 @@ module Convection
 
       private
 
-      def emit_events(event, *errors)
+      def task(task_name, stack)
```

Comment:
This gets confusing when you look at stack tasks such as the ones that are being extended in #207.

How about `#operation` or something else?